modules/exploits/multi/script/web_delivery.rb

```diff
@@ -103,6 +105,10 @@ def initialize(info = {})
 
   register_advanced_options(
     [
+      OptBool.new('Powershell::exec_in_place', [true, 'Produce PSH without executable wrapper', true]),
```

Comment:
Are these Powershell:: options really necessary here? If you want to override the default we can do something like: https://github.com/phra/metasploit-framework/pull/1

Reply:
fixed.